.github/dependabot.yml

```diff
@@ -4,3 +4,7 @@ updates:
   directory: "/"
   schedule:
     interval: "weekly"
+ - package-ecosystem: "devcontainers"
+   directory: "/"
+   schedule:
+     interval: weekly
```

Comment:
```suggestion
- package-ecosystem: "devcontainers"
  directory: "/"
  schedule:
    interval: weekly
```

Reply:
ha ha - I spotted those tabs and I believe they might be tabs in the existing file